blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -0,0 +1,295 @@
+
+To use the new help feature, make sure you are logged in as administrator, then click on the question mark in the upper right corner.
+Please note that it is only shown if your screen is wide enough (>= 1280 px).
+
+### UI Command Item
+
+I guess since the introduction of Main UI with openHAB 3.0, which was released exactly three years ago (let us ignore that one day difference), many users have moved over their home automation dashboards from HABPanel to Main UI.
+But the whole time, Main UI was lacking a feature, that HABPanel has: The ability to control the UI using an Item.
+
+In HABPanel, it is possible to switch the dashboard by commanding an Item - what if you could do the same (and much more) in Main UI?
+openHAB 4.1 finally adds that feature!
+
+The UI Command Item basically is a `String` Item to which these supported commands are sent:
+
+- `navigate`: Navigate to a given page path, e.g. `navigate:/locations` to navigate to the locations tab.
+- `popup`, `popover` & `sheet`: Open a widget, an `oh-` component or a page in the given modal, e.g. `popup:oh-clock-card`.
+- `notification`: Display a notification inside the UI, e.g. `notification:Merry Christmas and happy holidays!:openHAB 4.1`
+- `close`: Close all popups, popovers and sheets.
+- `back` & `reload`: Navigate back or reload the current page.
+
+The UI Command Item has to be configured for each client inside Main UI's settings: "Help & About" > "Listen for UI commands to".
+Unfortunately, we don`t have documentation for that at the moment, but for those wanting to play around with that new feature, you can have a look at the [pull request](https://github.com/openhab/openhab-webui/pull/2055) for some additional examples.
+
+Many thanks to Dan Cunningham ([@digitaldan](https://github.com/digitaldan)) for implementing that!
+
+## Sitemap and Basic UI Enhancements
+
+_Laurent Garnier, openHAB Maintainer_
+
+### New Sitemap element: `Buttongrid`
+
+A `Buttongrid` represents a grid of buttons as typical for remote controls:
+It allows to arrange a number of buttons, either labelled with a string or an icon, in a custom format.
+When a button is pressed, the associated command is sent to the linked Item.
+
+Here is the result in Basic UI, using the following definition:
+
+```java
+Buttongrid label="Remote Control" staticIcon=screen item=RemoteControl buttons=[1:1:POWER="Power"=switch-off , 1:2:MENU="Menu" , 1:3:EXIT="Exit" , 2:2:UP="Up"=f7:arrowtriangle_up , 4:2:DOWN="Down"=f7:arrowtriangle_down , 3:1:LEFT="Left"=f7:arrowtriangle_left , 3:3:RIGHT="Right"=f7:arrowtriangle_right , 3:2:OK="Ok" , 2:4:VOL_PLUS="Volume +" , 4:4:VOL_MINUS="Volume -" , 3:4:MUTE="Mute"=soundvolume_mute]
+```
+
+<p align="center"><img style="max-width: 70%;" src="/uploads/2023-12-22-openhab-4-1-release/basicui-buttongrid.png"/></p>
+
+Please note that this new element is currently only supported in Basic UI, but it is already worked on supporting it in the Android app as well.
```

Comment:
```suggestion
Please note that the support for this new element is currently limited to Basic UI. However, efforts are underway to extend its support to the Android app.
```

(or similar, the wording "is is already worked on supporting" seems a bit complex)